pkg/controller/core.oam.dev/v1alpha2/application/assemble/assemble.go

```diff
@@ -300,6 +301,11 @@ func (am *AppManifests) filterAndSetAnnotations(obj *unstructured.Unstructured)
 }
 
 func (am *AppManifests) setNamespace(obj *unstructured.Unstructured) {
```

Comment:
please add comment like “setNamespace set namespace except for Namespace object"

Reply:
to make it clearer for other developer

Reply:
got it, thanks